rxnetty-spectator-tcp/src/main/java/io/reactivex/netty/spectator/LatencyMetrics.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2015 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.reactivex.netty.spectator;
+
+import com.netflix.numerus.NumerusProperty;
+import com.netflix.numerus.NumerusRollingPercentile;
+import com.netflix.spectator.api.ExtendedRegistry;
+import com.netflix.spectator.api.Id;
+import com.netflix.spectator.api.Spectator;
+import com.netflix.spectator.api.ValueFunction;
+
+import java.util.concurrent.TimeUnit;
+
+import static com.netflix.numerus.NumerusProperty.Factory.*;
+
+/**
+ * A latency metric for publishing various percentiles of latency which spectator does not.
+ */
+public class LatencyMetrics {
+
+    private static final NumerusProperty<Integer> latency_timeInMilliseconds = asProperty(60000);
+    private static final NumerusProperty<Integer> latency_numberOfBuckets = asProperty(12); // 12 buckets at 5000ms each
+    private static final NumerusProperty<Integer> latency_bucketDataLength = asProperty(1000);
```

Comment:
@adriancole are you asking, what does it do if there are more samples than the bucket data length?
If so, then it drops the old data, relevant code is [here](https://github.com/Netflix-Skunkworks/Numerus/blob/master/numerus/src/main/java/com/netflix/numerus/NumerusRollingPercentile.java#L286)